content/en/developers/guide/unified-tagging-advanced-usage.md

```diff
@@ -0,0 +1,74 @@
+
+```yaml
+tags.datadoghq.com/<container>.env
+tags.datadoghq.com/<container>.service
+tags.datadoghq.com/<container>.version
+```
+
+## What if I want/have different tags for env/service/version?
+You can continue to use these tags as you have them configured today. We have standardized on env, service, and version to make tagging consistent and simple. If you'd like to achieve a unified tagging experience with your own custom tags, there are some options available to you.
+
+But note that while some products support arbitrary tags, others have more specific expectations. As such, navigation between products might be difficult if one data source has a tag that another does not have or does not support.
+
+Why do I see a different version in my spans if I've set DD_VERSION?
+You might be hard-coding the version tag in your spans. To address this issue, rename the span tag to something else (e.g., payload_version). That way it won't conflict with setting the service's version for the version tag.
+
+### Containerized environments
+
+The standard labels in containerized environments are automatically mapped to their plain tag equivalents. But through Autodiscovery this are [broader tagging options][1] available, whether it's defining a list of tags to associate with a container or mapping labels/annotations to specific tags.
+
+#### Metrics
+
+Since tags can be appended to data points, you have a lot of freedom in setting desired tags. Autodiscovered tags are automatically added to all metrics collected.
+
+#### APM
+
+`env` and `service` are core tags in APM, so it is not possible to replace them with differently named tags. However, APM does allow for data to be [aggregated along more primary tags] than just `env`. Since host tags are added to traces and trace metrics, they can be used as well (e.g., availability-zone).
+
+Trace metrics also have a specific [whitelist of tags]. Container tags are not included.
+
+Autodiscovered tags associated with containers are added to `container_info` in span metadata.
+
+#### Logs
+Similar to APM, service is a core tag that is used to help organize logs data. Additionally, it is not possible to link from a log to the related APM service without it.
+
+Similar to metrics however, Autodiscovered tags for the container as well as host tags for the agent are added to all logs.
+
+You also have the ability to add custom fields to your logs in-code, which can be mapped to tags or attributes downstream in a [Datadog log processing pipeline].
+
+### Non-containerized
+
+While host tags can be used to conveniently add service to all data sources, we recommend against setting it as a host tag - especially if you have multiple APM services that run on a host.
+
+We'd recommend taking advantage of each product's options for tagging.
```

Comment:
@apigirl @cswatt 

If we keep these sections to talk about options for tagging, then we'll want to include links to the appropriate docs by product.

Although the direction/guidance is largely the same across containerized and non-containerized. The only big difference is that we don't have autodiscovery in the latter.

Should we re-org to discuss containerized vs. non-containerized and then have individual sections by product?